mne/io/artemis123/artemis123.py

```diff
@@ -32,6 +35,12 @@ def read_raw_artemis123(input_fname, preload=False, verbose=None):
         on the hard drive (slower, requires less memory).
     verbose : bool, str, int, or None
         If not None, override default verbose level (see mne.verbose).
+    pos_fname : str or None (defulat None)
+        If not None, load digitized head points from this file
+    head_loc : bool (default False)
+        If True attempt to perform head localization using HPI coils. If no
+        HPI coils are in info['dig'] hpic coils are assumed to be in canonical
+        order of fiducial points (nas, rpa, lpa).
```

Comment:
So I pushed some changes. let me know what you think.

I think leaving it as `head_loc` makes sense. it is performing HPI localization and then setting `info['dev_head_t]`, `add_trans` while technically correct isn't really meaningful.

It defaults to `True` because the artemis scanner doesn't perform any of these computations so the file doesn't have a `dev_head_t`. So in order to have a meaningful head coordinate system you need to do head_localization. 

I think it fails gracefully. If less then 3 chpi coils are on it doesn't issues a warning and sets the `dev_head_t` to identity. If you don't supply and digitization points and also get the ordering wrong (You plugged the coils into the wrong physical ports) then there is no way to catch that error. You'll get a `dev_head_t` that is incorrect.